.github/workflows/ruby-ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Ruby-CI
+
+on:
+  push:
+  branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  ruby-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout
+
+      - uses: ruby/setup-ruby@v1
```

Comment:
Using `ruby/setup-ruby` action over `actions/setup-ruby` since actions/setup-ruby [will be deprecated](https://github.com/actions/setup-ruby/issues/80)